api/admin/routes.py

```diff
@@ -23,10 +23,15 @@
 import csv, codecs, cStringIO
 from StringIO import StringIO
 import urllib
+from datetime import timedelta
 
 # The secret key is used for signing cookies for admin login
 app.secret_key = Configuration.get(Configuration.SECRET_KEY)
 
+# An admin's session will expire after this amount of time and
+# the admin will have to log in again.
+app.permanent_session_lifetime = timedelta(hours=4)
```

Comment:
I would set it at at 9 hours so a librarian doesn't have to log in again during their workday.